tests/chart_tests/test_houston_api_deployment.py

```diff
@@ -21,15 +22,39 @@ def test_houston_api_deployment(self, kube_version):
         doc = docs[0]
         assert doc["kind"] == "Deployment"
         assert "annotations" not in doc["metadata"]
+        assert {
+            "tier": "astronomer",
+            "component": "houston",
+            "release": "release-name",
+        } == doc["spec"]["selector"]["matchLabels"]
+
         assert (
-            {
-                "tier": "astronomer",
-                "component": "houston",
-                "release": "release-name",
-            }
-            == doc["spec"]["selector"]["matchLabels"]
-            == doc["spec"]["template"]["metadata"]["labels"]
+            "app" in doc["spec"]["template"]["metadata"]["labels"]
+            and "houston" in doc["spec"]["template"]["metadata"]["labels"]["app"]
         )
+        assert (
+            "tier" in doc["spec"]["template"]["metadata"]["labels"]
+            and "astronomer" in doc["spec"]["template"]["metadata"]["labels"]["tier"]
+        )
+        assert (
+            "component" in doc["spec"]["template"]["metadata"]["labels"]
+            and "houston" in doc["spec"]["template"]["metadata"]["labels"]["component"]
+        )
+        assert (
+            "release" in doc["spec"]["template"]["metadata"]["labels"]
+            and "release-name"
+            in doc["spec"]["template"]["metadata"]["labels"]["release"]
+        )
+
+        # assert (
+        #         {
+        #             'app': 'houston',
+        #             "tier": "astronomer",
+        #             "component": "houston",
+        #             "release": "release-name",
+        #         }
+        #         in doc["spec"]["template"]["metadata"]["labels"]
+        # )
```

Comment:
Uncommenting this and doing the tests this way might be better, but dealing with the `version` label would be problematic so we'd need to find a way to skip that one.

Reply:
This would work

```suggestion
        labels = doc["spec"]["template"]["metadata"]["labels"]
        assert {'tier': 'astronomer', 'component': 'houston', 'release': 'release-name', 'app': 'houston'} == {x: labels[x] for x in labels if x != "version"}
```